### PR TITLE
Fix ReactARTStack bundles to not include DOM Stack inside

### PR DESCRIPTION
**Pull request by gaearon**
Only affects our internal FB bundles.

### CODE REVIEW

File: scripts/rollup/modules.js [hunk 1]
```diff
@@ -171,13 +171,6 @@ function getInternalModules() {
   };
 }
 
-function replaceInternalModules() {
-  // we inline these modules in the bundles rather than leave them as external
-  return {
-    "'react-dom'": `'${resolve('./src/renderers/dom/ReactDOM.js')}'`,
```
Comment (gaearon) [inline scripts/rollup/modules.js]: Rather than have this special case in all bundles, I opted for importing `ReactDOM` explicitly in test utils.

**Comment by gaearon** [comment]
This doesn't matter much but makes ART bundle smaller for test cases (where we still use it). Which I guess is a minor benefit. But it also removes the `react-dom` aliasing hack which is confusing.

**Comment by acdlite** [comment]
This isn't working for me in master. If I run `yarn build -- --type=FB` I still get `require('react-dom')` in ART and test renderer builds.

**Comment by trueadm** [comment]
@acdlite @gaearon  this may be because `'react-dom` is specified in the externals for the Fiber bundle (`scripts/rollup/bundles.js`)?

**Comment by acdlite** [comment]
Ah I read the title of this PR incorrectly and assumed it was related. Thanks!

**Pull request merged**